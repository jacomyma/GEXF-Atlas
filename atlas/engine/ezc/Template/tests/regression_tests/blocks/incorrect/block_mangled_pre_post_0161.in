{* file: non_matching_block_0161.in *}
{_reseta}

