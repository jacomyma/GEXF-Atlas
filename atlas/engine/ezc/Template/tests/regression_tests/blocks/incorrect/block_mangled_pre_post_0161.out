mock:2:2: Unknown block '_reseta'.

{_reseta}
 ^
